lib/cli/package.json

```diff
@@ -35,8 +35,9 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/ejs": "^3.1.0",
-    "@types/shelljs": "^0.8.7",
-    "typescript": "3.9.3"
+    "@types/ejs": "^3.1.1",
+    "@types/node": "^18.13.0",
+    "@types/shelljs": "^0.8.11",
+    "typescript": "^4.9.5"
```

Comment:
Have you checked the list of [breaking changes](https://github.com/microsoft/TypeScript/wiki/Breaking-Changes) when migrating from TypeScript 3 to 4?

Reply:
as soon as it compiles the CLI it should be fine :)